Code/Framework/AzCore/AzCore/Serialization/EditContext.cpp

```diff
@@ -41,6 +42,28 @@ namespace AZ
         m_enumData.clear();
     }
 
+    //=========================================================================
+    // EditContext::EnumerateInstanceCallContext
+    //=========================================================================
+    EditContext::EnumerateInstanceCallContext::EnumerateInstanceCallContext(
+        const SerializeContext::BeginElemEnumCB& beginElemCB,
+        const SerializeContext::EndElemEnumCB& endElemCB,
+        const EditContext* context,
+        unsigned int accessFlags,
+        SerializeContext::ErrorHandler* errorHandler)
+        : m_beginElemCB(beginElemCB)
+        , m_endElemCB(endElemCB)
+        , m_accessFlags(accessFlags)
+        , m_context(context)
+    {
+        m_errorHandler = errorHandler ? errorHandler : &m_defaultErrorHandler;
+
+        m_elementCallback = [this](void* ptr, const Uuid& classId, const SerializeContext::ClassData* classData, const SerializeContext::ClassElement* classElement) -> bool
+        {
+            return m_context->EnumerateInstance(this, ptr, classId, classData, classElement);
+        };
+    }
```

Comment:
Instead of storing a const copy of the EditContext*, the EditContext can bound as a member of the lambda.
```suggestion
        m_elementCallback = [this, context](void* ptr, const Uuid& classId, const SerializeContext::ClassData* classData, const SerializeContext::ClassElement* classElement) -> bool
        {
            return context->EnumerateInstance(this, ptr, classId, classData, classElement);
        };
    }
```